src/pages/trade-types/margin/_available-markets.tsx

```diff
@@ -82,18 +81,13 @@ const AvailableMarkets = () => {
                                     />
 
                                     <StyledText weight="bold">
```

Comment:
```suggestion
const StyledText = styled(Header)`
    margin-top: 1.6rem;
    @media ${device.mobileL} {
        font-size: 18px;
        margin-top: 0;
    }
`
                                    <StyledText as="div">
```

`Text` is deprecated and `Header` always bold be default, let's refactor it a little bit here and below